### PR TITLE
Add a navigation bar

### PR DESCRIPTION
**Pull request by KyleQ1**
Add a simple navbar to make the transition throughout the site easier.

### CODE REVIEW

**Comment by calcmogul** [comment]
This needs to be rebased to fix CI.

File: admin/goals/index.html [hunk 1]
```diff
@@ -9,6 +9,17 @@
...
 <body>
+  <header id="navbar">
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="admin/">Admin</a></li>
+      <li><a href="bootstrap/">Developer Bootstrap</a></li>
+      <li><a href="vpn/">VPN Setup</a></li>
+      <li><a href="ci/">Curriculum</a></li>
+      <li><a href="fll/">FLL</a></li>
+      <li><a href="archives/">Archives</a></li>
```
Comment (calcmogul) [inline admin/goals/index.html:20]: These relative hrefs are incorrect. For example, `admin/` will go to `https://frc3512.github.io/admin/goals/admin`. The same problem applies to all the others. You'll need to prefix them with `../`.

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: ci/github/index.html [hunk 2]
```diff
@@ -140,12 +165,17 @@ <h3 id="New_patchset">New patchset</h3>
...
 
 Fixes #42.</pre>
     <p>where <code>42</code> would be the issue number being closed by this PR.
     The last line ensures the issue is automatically closed when the PR is
-    merged. Make sure each issue resolution is a separate branch and PR.</p>
+    ======= Optional extended description that should be complete sentences.
+    Fixes #number.</p>
+    <p>The last line ensures the issue is automatically closed when the PR is
+    &gt;&gt;&gt;&gt;&gt;&gt;&gt; Stashed changes merged. Make sure each issue
+    resolution is a separate branch and PR.</p>
```
Comment (calcmogul) [inline ci/github/index.html:178]: Something weird happened here during a rebase. This should get cleaned up.

File: main.css [hunk 3]
```diff
@@ -36,7 +36,6 @@ code {
 div.body {
   margin: 2em auto;
   max-width: 38.75em;
-  padding: 1em;
```
Comment (calcmogul) [inline main.css:39]: What's the reason for removing this?

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: main.css (hunk 3 above)
Comment (KyleQ1) [reply main.css:39]: There's a lot of whitespace at the top of each webpage which makes the web page look weird.

File: ci/github/index.html [hunk 4]
```diff
@@ -139,8 +164,6 @@ <h3 id="New_patchset">New patchset</h3>
     upstream and lead to conflicts. Commit messages should have a format like
     this:</p>
     <pre>Implement a thing
-    
-Optional extended description that should be complete sentences.
```
Comment (calcmogul) [inline ci/github/index.html:143]: I don't think this should be removed. It's intended to show the format of a commit message.

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: ci/github/index.html (hunk 4 above)
Comment (KyleQ1) [reply ci/github/index.html]: I didn't mean to remove that

**Review by calcmogul — APPROVED** [review]

**Pull request merged**